packages/web/src/common/services/audius-backend/AudiusBackend.ts

```diff
@@ -94,6 +94,7 @@ declare global {
     phantom: any
     solana: PhantomProvider
     web3: any
```

Comment:
what even is lowercase web3? I feel like that doesn't exist actually

Reply:
yeah agreed, this looks like a bug